guessit/rules/common/quantity.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Quantity
+"""
+import re
+
+import six
+
+
+class Quantity(object):
```

Comment:
I'm back from holidays ☺️. base class + subclasses with explicit names seems to be better imho.